.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should add this file exclusion to the `.gitignore` for the project, as this is a per-user sort of thing.